doxia-site-renderer/src/main/resources/org/apache/maven/doxia/siterenderer/resources/default-site.vm

```diff
@@ -17,7 +17,7 @@
 ## under the License.
 ##
 #parse( "default-site-macros.vm" )
-<!-- Generated by Apache Maven Doxia Site Renderer#if( $doxiaSiteRendererVersion ) $doxiaSiteRendererVersion#end##
+<!-- Generated by #generator($doxiaSiteRendererVersion)
```

Comment:
This name is inconsistent with apache/maven-fluido-skin#9. Please unify.

Reply:
Akin to 57405f71ea3d38840f9711fb9b2c02f5033eb768.

Reply:
I unified with the merged result in the apache/maven-fluido-skin master (indeed passing a parameter was redundant, my bad).